.gitlab/pkg_metrics/pkg_metrics.yml

```diff
@@ -2,197 +2,112 @@
-    - AMD64_RPM_AGENT_SIZE=$(du -sB1 /tmp/rpm/amd64-agent | sed 's/\([0-9]\+\).\+/\1/')
-    # centos arm64
-    - cd /tmp/rpm/arm64-agent && rpm2cpio $OMNIBUS_PACKAGE_DIR/datadog-agent-6.*.aarch64.rpm | cpio -idm > /dev/null
-    - ARM64_RPM_AGENT_SIZE=$(du -sB1 /tmp/rpm/arm64-agent | sed 's/\([0-9]\+\).\+/\1/')
-
-    # suse
-    - cd /tmp/suse/amd64-agent && rpm2cpio $OMNIBUS_PACKAGE_DIR_SUSE/datadog-agent-6.*.x86_64.rpm | cpio -idm > /dev/null
-    - AMD64_SUSE_AGENT_SIZE=$(du -sB1 /tmp/suse/amd64-agent | sed 's/\([0-9]\+\).\+/\1/')
-
-    - currenttime=$(date +%s)
-    - DD_API_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $API_KEY_SSM_NAME)
-    - |
-      curl --fail -X POST -H "Content-type: application/json" \
-      -d "{\"series\":[
-            {\"metric\":\"datadog.agent.package.size\",\"points\":[[$currenttime, $AMD64_DEB_AGENT_SIZE]], \"tags\":[\"os:debian\", \"package:agent\", \"agent:6\", \"bucket_branch:$BUCKET_BRANCH\", \"arch:amd64\"]},
-            {\"metric\":\"datadog.agent.package.size\",\"points\":[[$currenttime, $AMD64_RPM_AGENT_SIZE]], \"tags\":[\"os:centos\", \"package:agent\", \"agent:6\", \"bucket_branch:$BUCKET_BRANCH\", \"arch:amd64\"]},
-            {\"metric\":\"datadog.agent.package.size\",\"points\":[[$currenttime, $AMD64_SUSE_AGENT_SIZE]], \"tags\":[\"os:suse\", \"package:agent\", \"agent:6\", \"bucket_branch:$BUCKET_BRANCH\", \"arch:amd64\"]},
-            {\"metric\":\"datadog.agent.package.size\",\"points\":[[$currenttime, $ARM64_DEB_AGENT_SIZE]], \"tags\":[\"os:debian\", \"package:agent\", \"agent:6\", \"bucket_branch:$BUCKET_BRANCH\", \"arch:arm64\"]},
-            {\"metric\":\"datadog.agent.package.size\",\"points\":[[$currenttime, $ARM64_RPM_AGENT_SIZE]], \"tags\":[\"os:centos\", \"package:agent\", \"agent:6\", \"bucket_branch:$BUCKET_BRANCH\", \"arch:arm64\"]}
-          ]}" \
-      "https://api.datadoghq.com/api/v1/series?api_key=$DD_API_KEY"
+    # Allow failures: some packages are not always built, and therefore stats cannot be sent for them
+    - set +e
 
-send_pkg_size-a7:
-  allow_failure: true
-  rules: !reference [.on_deploy_a7]
-  stage: pkg_metrics
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
-  tags: ["arch:amd64"]
-  needs:
-    - agent_deb-x64-a7
-    - agent_deb-arm64-a7
-    - iot_agent_deb-x64
-    - iot_agent_deb-arm64
-    - dogstatsd_deb-x64
-    - dogstatsd_deb-arm64
-    - agent_heroku_deb-x64-a7
-    - agent_rpm-arm64-a7
-    - agent_rpm-x64-a7
-    - iot_agent_rpm-x64
-    - iot_agent_rpm-arm64
-    - dogstatsd_rpm-x64
-    - agent_suse-x64-a7
-    - dogstatsd_suse-x64
-    - iot_agent_suse-x64
-  before_script:
-    # FIXME: tmp while we uppdate the base image
-    - apt-get install -y wget rpm2cpio cpio
     - ls -l $OMNIBUS_PACKAGE_DIR
     - ls -l $OMNIBUS_PACKAGE_DIR_SUSE
```

Comment:
This is mainly for debug purposes, to "see" what artifacts the job did manage to download. I'd keep it as-is because it's not harmful, and has a small chance to be useful for investigation if this job fails unexpectedly.